runtime/src/apis.rs

```diff
@@ -248,7 +248,7 @@ impl_runtime_apis! {
 			use frame_benchmarking::{BenchmarkError, Benchmarking, BenchmarkBatch};
 			use super::*;
 
-			use frame_system_benchmarking::Pallet as SystemBench;
+			//use frame_system_benchmarking::Pallet as SystemBench;
```

Comment:
Was this removed for testing?

Reply:
Noted, will check on it

Reply:
It was declared but never used.  I will restore it back